test/index.js

```diff
@@ -24,4 +24,18 @@ describe('PrismJS Configuration', () => {
             assert.equal(actual.trim(), expected.trim());
         });
     });
+
+    it(`should work with all languages`, () => {
```

Comment:
I didn't create a test in the `fixtures` folder because we literally load all languages. This means that the languages loaded heavily depend on the current Prism version. So instead of saying "It has to import these languages in that order", I just check that it imports a large number of languages.